### PR TITLE
feat: make dcl install

### PR DESCRIPTION
**Pull request by leamendoza97**
# What? 
Add the command **install** to install decentraland libraries and to check all decentraland libraries installed which need to be in bundleDependencies field of _package.json_

- [x] Main functionality, look for all dependencies and select decentralandLibraries
- [x] Install a new package
- [x] Tests

# Why? 
See #567

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ const help = `
       init                Create a new Decentraland Scene project
       build               Build scene
       start               Start a local development server for a Decentraland Scene
+      install             Install a package and/or sync bundleDependencies
```
Comment (leamendoza97) [inline src/main.ts:41]: Maybe two lines with 
- install
- install package
?

**Review by menduz — APPROVED** [review]
Awesome, only one question. Are the `␊` in the new lines required?

**Comment by leamendoza97** [comment]
> Awesome, only one question. Are the `␊` in the new lines required?

That file is auto-generated by the `npm run test:dry` to update the snapshots which match in the CI running so, I think it's okay.

**Comment by menduz** [comment]
🚢 it

**Pull request merged**